### PR TITLE
basic warnings for nonsense functions

### PR DESCRIPTION
**Pull request by BMaxV**
There are issues with DirectGUI, to fix them, let's introduce some warnings and remove code later.

### CODE REVIEW

File: direct/src/gui/OnscreenGeom.py [hunk 1]
```diff
@@ -111,6 +113,7 @@ def setGeom(self, geom,
                 self.setColor(color[0], color[1], color[2], color[3])
 
     def getGeom(self):
+        warnings.warn("stop using this function",FutureWarning)
```
Comment (rdb) [inline direct/src/gui/OnscreenGeom.py]: Style-wise, there should be a space after the comma.  We could probably improve the wording of the warning message, like "This method is vestigial and will be removed shortly."

**Comment by Moguri** [comment]
@rdb were you waiting on CI (i.e., existing tests) to verify that these methods are vestigial or are there new tests that should be created?

**Comment by rdb** [comment]
Hmm, how can we automate testing that effectively—perhaps it is possible to configure pytest that it sees warnings as errors?

**Comment by Moguri** [comment]
@BMaxV, Are you still interested in pursuing this PR? If not, or if I don't here from you in two weeks, I intend to close this PR. It can always be re-opened.

**Comment by BMaxV** [comment]
Yes I am still interested in this PR.

**Comment by Moguri** [comment]
@BMaxV glad to hear that! Do we have some next steps for this PR?

**Comment by Moguri** [comment]
@rdb and @BMaxV since these functions are actually being used, should we close this PR?

**Pull request closed without merging**